<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PosTestWindow</class>
 <widget class="QMainWindow" name="PosTestWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>845</width>
    <height>656</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>10</y>
      <width>401</width>
      <height>311</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="listAndExitButton">
     <property name="sizeConstraint">
      <enum>QLayout::SetFixedSize</enum>
     </property>
     <item>
      <widget class="QPushButton" name="m_button">
       <property name="text">
        <string>Button</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>370</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Button position</string>
    </property>
   </widget>
   <widget class="QLabel" name="m_buttonX">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>390</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>X = </string>
    </property>
   </widget>
   <widget class="QLabel" name="m_buttonY">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>410</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Y = </string>
    </property>
   </widget>
   <widget class="QLabel" name="m_buttonGlobalX">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>440</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Global X =</string>
    </property>
   </widget>
   <widget class="QLabel" name="m_buttonGlobalY">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>460</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Global Y = </string>
    </property>
   </widget>
   <widget class="QPushButton" name="m_update">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>490</y>
      <width>399</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Update</string>
    </property>
   </widget>
   <widget class="QLabel" name="m_buttonParentY">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>410</y>
      <width>231</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Parent Y = </string>
    </property>
   </widget>
   <widget class="QLabel" name="m_buttonParentX">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>390</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Parent X = </string>
    </property>
   </widget>
   <widget class="QLabel" name="m_buttonWindowX">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>440</y>
      <width>261</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Window X = </string>
    </property>
   </widget>
   <widget class="QLabel" name="m_buttonWindowY">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>460</y>
      <width>271</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Window Y = </string>
    </property>
   </widget>
   <widget class="QPushButton" name="enableButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>390</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Enable button</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="m_enableCheckBox">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>440</y>
      <width>71</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>CheckBox</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="m_enableRadioButton">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>470</y>
      <width>82</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>RadioButton</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="m_enableLineEdit">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>500</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>181</width>
      <height>91</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>181</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>845</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuItems">
    <property name="title">
     <string>Items</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuItems"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
